.github/workflows/auto-label.yml

```
@@ -82,20 +82,20 @@ jobs:
 
-              // Process package changes
-              if (changedPackages && changedPackages.trim()) {
-                // Add specific package labels
-                const packages = changedPackages.trim().split(/\s+/).filter(Boolean);
-                packages.forEach(pkg => labelsToAdd.add(pkg));
+              // Process workspace changes
+              if (changedWorkspaces && changedWorkspaces.trim()) {
+                // Add specific workspace labels
+                const workspaces = changedWorkspaces.trim().split(/\s+/).filter(Boolean);
+                workspaces.forEach(ws => labelsToAdd.add(ws));
                 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)